TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -104,14 +104,14 @@ public function initializeArguments()
         $this->registerTagAttribute('alt', 'string', 'Specifies an alternate text for an image', true);
         $this->registerTagAttribute('ismap', 'string', 'Specifies an image as a server-side image-map. Rarely used. Look at usemap instead', false);
         $this->registerTagAttribute('usemap', 'string', 'Specifies an image as a client-side image-map', false);
-        // @deprecated since 2.0 use the "image" argument instead
-        $this->registerArgument('asset', 'TYPO3\Media\Domain\Model\AssetInterface', 'The image to be rendered - DEPRECATED, use "image" argument instead', false);
+        // @deprecated since 2.1 use "asset" argument instead
+        $this->registerArgument('image', 'TYPO3\Media\Domain\Model\ImageInterface', 'The image to be rendered - DEPRECATED, use "asset" argument instead', false);
```

Comment:
doubt many actually did that to be honest and since there's a good reason for both deprecations (even though they are contradicting) and the backward compatibility I don't see it as an issue tbh. Implementing two thumbnail view helpers feels wrong to me, was like that earlier and didn't like it. Not doing it now doesn't prevent doing so in the future if we see the need for it.

One problem with such a thumbnail view helper is that you'd have to ensure you're working with images, which requires php to achieve currently.